webpack.config.js

```diff
@@ -171,12 +170,12 @@ const webpackConfig = {
 	},
 	optimization: {
 		splitChunks: {
-			chunks: codeSplit ? 'all' : 'async',
+			chunks: ! isDesktop ? 'all' : 'async',
 			name: !! ( isDevelopment || shouldEmitStats ),
 			maxAsyncRequests: 20,
 			maxInitialRequests: 5,
 		},
-		runtimeChunk: codeSplit ? { name: 'manifest' } : false,
+		runtimeChunk: ! isDesktop ? { name: 'manifest' } : false,
```

Comment:
`splitChunks.chunks` and `runtimeChunk` could have the same configuration for desktop and non-desktop. I don't see any reason to make them different.

Reply:
It just requires a bit of rework to the desktop.jsx landing page... Maybe not be a big deal, already had to add some bits in there in https://github.com/Automattic/wp-calypso/pull/33585/commits/f12d1b2d12190d41d8af67d3be8e208cc4225315